k8s/agent-startup-lease.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: logdna-agent-startup-lease
+  namespace: logdna-agent
+rules:
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
+  verbs: ["get", "watch", "list", "update", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: logdna-agent-startup-lease
+  namespace: logdna-agent
+subjects:
+- kind: ServiceAccount
+  name: logdna-agent
+  namespace: logdna-agent
+roleRef:
+  kind: Role
+  name: logdna-agent-startup-lease
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: coordination.k8s.io/v1
+kind: LeaseList
+items:
+  - apiVersion: coordination.k8s.io/v1
+    kind: Lease
+    metadata:
+      name: logdna-agent-startup-lease-0
+      namespace: logdna-agent
+      labels:
+        process: logdna-agent-startup
```

Comment:
We had to remove the logdna- prefix for the process value. Is it something to be fixed?